src/main/java/data/Account.java

```diff
@@ -196,4 +196,20 @@ public String buildJsonConfig() {
       getCurrency(), getDirection(), getEntity(), getMatchingMode(), 
       getAccountID(), getAggregationMode());
   }
+
+  /**
+   * @param request contains form data to edit existing account
+   */
+  public void updateExistingAccount(HttpServletRequest request) {
```

Comment:
This looks identical to the existing Account(HttpServletRequest) constructor. Is this needed?

Reply:
+1